src/multi_vector_simulator/D0_modelling_and_optimization.py

```diff
@@ -269,6 +264,15 @@ def simulating(dict_values, model, local_energy_system):
         """
         Initiates the oemof-solph simulation, accesses results and writes main results into dict
 
+        If an error is encountered in the oemof solver, mvs should not be allowed to continue,
+        otherwise other errors related to the uncomplete simulation result might occur and it will
+        be more obscure to the endusers what went wrong.
+
+        A MVS error is raised if the omoef solver warning states explicitely that
+        "termination condition infeasible", otherwise the oemof solver warning is re-raised as
+        an error.
+
+
```

Comment:
Good idea to circumvent it - maybe we should still always write the full oemof/pyomo error message to the log file?

Reply:
it is the case already on line 309

Reply:
> Does this already include that the error message is forwarded to the EPA?

This is not included in this PR as the MVS server is already returning to the EPA the traceback of any exception raised during the parsing, simulation or analysis of the results